tests/network/vmi_multus.go

```diff
@@ -423,7 +444,7 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 					libvmi.WithNetwork(v1.DefaultPodNetwork()),
 					libvmi.WithInterface(linuxBridgeInterface),
 					libvmi.WithNetwork(&linuxBridgeNetwork),
-					libvmi.WithCloudInitNoCloudNetworkData(cloudInitNetworkDataWithStaticIPsByDevice("eth1", "10.1.1.2/24"), false))
+					libvmi.WithCloudInitNoCloudNetworkData(cloudInitNetworkDataWithStaticIPsByDevice("eth1", fmt.Sprintf("%s/24", ptpSubnetIP2)), false))
```

Comment:
The length of the prefix is not something known to the test. Could we define it as one of the constants where it is right next to the cidr?

Reply:
Done